3 Reports.sql

```diff
@@ -0,0 +1,27 @@
+Use PetDB
+go
+
+Reports:
+--1)We need a list of how many of each pet we currently have.
+SELECT p.PetType, Amount = count(*)
+from Pet p 
+Where p.active = 1
+Group by p.PetType
+
+--2)we need to see how many customers have multiple pets.
+SELECT NumOfCustomers = Count(*) from (
```

Comment:
This can be done simply with a group by and having, no need for a subquery.

Reply:
When I do a simple group by/having, I get a list of the costumers who have more than 1 pet, which, to be honest, is more useful information, but the biz-scenario asks for "how many customers have multiple pets" not a "list of customers that have multiple pets". Am I missing something? Is there a way to get the count of customers who have multiple pets without a subquery?

Reply:
The question was asking that, show a list of customers that have multiple pets, sorry it wasn't so clear. Please fix